crates/bin/pindexer/src/indexer_ext.rs

```diff
@@ -12,7 +12,14 @@ impl IndexerExt for cometindex::Indexer {
             .with_index(Box::new(crate::stake::DelegationTxs {}))
             .with_index(Box::new(crate::stake::UndelegationTxs {}))
             .with_index(Box::new(crate::governance::GovernanceProposals {}))
-            .with_index(Box::new(crate::dex_ex::Component::new()))
+            .with_index(Box::new(crate::dex_ex::Component::new(
+                penumbra_asset::asset::Id::from_str(
+                    // USDC
+                    "passet1w6e7fvgxsy6ccy3m8q0eqcuyw6mh3yzqu3uq9h58nu8m8mku359spvulf6",
```

Comment:
This asset id is only relevant to the chain `penumbra-1`, no? How will the indexing work on e.g. `penumbra-testnet-phobos-2`? Hard to evaluate the behavior without having already updated the `dex-explorer` code for the testnet deployment, which is still unplanned.

Reply:
The same issue is also present for the insights dashboard ; maybe we should add a command line argument to override USDC? (in both cases)

Reply:
Not a bad idea, filed an issue in this repo https://github.com/penumbra-zone/penumbra/issues/4920